src/Util/DataStore.php

```diff
@@ -140,17 +142,17 @@ public function addFile(string $fileUri, string $datasetId, string $type)
         if (preg_match('/^http/', $fileUri)) {
             // Decode any characters escaped in the URL.
             $fileName = urldecode($fileName);
-            $browser = new \Buzz\Browser();
-            $result = $browser->head($fileUri);
-            $status = $result->getHeaders()[0];
-            if (!preg_match('/200/', $status)) {
+            $client = new GuzzleClient();
+            $result = $client->request('HEAD', $fileUri);
+            $status = $result->getStatusCode();
+            if (200 !== $status) {
                 throw new \Exception("File could not be downloaded from $fileUri ($status)");
             }
-            $contentType = $result->getHeader('Content-Type');
+            $contentType = $result->getHeader('Content-Type')[0];
             if (preg_match('#^text/html#', $contentType)) {
                 throw new HtmlFoundException("HTML file found at $fileUri");
             }
-            $contentDisposition = $result->getHeader('Content-Disposition');
+            $contentDisposition = $result->getHeader('Content-Disposition')[0];
```

Comment:
This one didn't exist at https://www.tamucc.edu/robots.txt
For available headers, see curl -I https://www.tamucc.edu/robots.txt